### PR TITLE
Enable webhooks when installing Halkyon

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #130

### CODE REVIEW

File: ansible/playbook/roles/halkyon/tasks/patch_master.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - name: Patch master-config file to use Mutating Webhook
+    command: "oc ex config patch {{ openshift_env.master_config_file }} --patch '{{ patch.stdout }}'"
+    register: r
+
+  - name: Copy patch to new master config file
+    copy: content="{{ r.stdout }}" dest={{ openshift_env.config_dir }}/master/master-config.yaml
+
+  - name: Restart master
+    include_role:
+      name: restart_master
```
Comment (cmoulliard) [inline ansible/playbook/roles/halkyon/tasks/patch_master.yml]: I dont think that we can use this role as cluster as been started using `oc cluster up` @geoand

Comment (geoand) [reply ansible/playbook/roles/halkyon/tasks/patch_master.yml]: Which cluster are we talking about?

Comment (geoand) [reply ansible/playbook/roles/halkyon/tasks/patch_master.yml]: I pushed  a change that should make it work with `oc cluster up` as well. 
The `openshift_env` role is the one that determines what kind of cluster is being used so theoretically it should work

Comment (cmoulliard) [reply ansible/playbook/roles/halkyon/tasks/patch_master.yml]: Ok. So this role will then perform a `docker restart origin` - https://github.com/snowdrop/openshift-infra/blob/master/ansible/playbook/roles/restart_master/tasks/main.yml#L14

Comment (geoand) [reply ansible/playbook/roles/halkyon/tasks/patch_master.yml]: Yeah, that's the idea

**Pull request merged**